src/containers/SignupSteps.js

```diff
@@ -13,15 +13,15 @@ import {
     setToken,
     setPrefix,
     setCompleted,
-    setTrackingId,
+    setTrackingId
 } from '../reducers/user';
 import { logCheckpoint } from '../reducers/tracking';
 import Signup from '../components/Signup';
 
 const mapStateToProps = (state, ownProps) => ({
     queryParams: ownProps.location.query,
     app: state.app,
-    user: state.user,
+    user: state.user
```

Comment:
these ones can stay actually; the eslint ruleset just doesn't want trailing commas in `import`.. not a big deal in any case